### ISSUE TITLE
Feature Request - [Download Progress]

### ISSUE THREAD

**Issue by Pesoen** [CypherpunkArmory/UserLAnd#568]
# Describe the problem your feature request solves.
When downloading many will assume its stuck downloading stuff(in my case its at 26/27 now and have been for the last 25 minutes)

this annoys me because i cannot tell if its stuck or doing something.

# Describe the solution you'd like 
Some kind of download progress readout(percentages done of the current file or more stats for nerds like Y mb downloaded out of X mb)

# Describe alternatives you've considered
nothing really since i cannot really get this kind of feature unless its requested.

# Additional context
nothing really, i think the above should be descriptive enough for now.

**Comment by MatthewTighe** [comment]
I'm going to continue to look at making the download process more robust, and can potentially look into displaying a more detailed progress bar during downloads.

For now, you can check download progress in your notification tray, as we use the Android Download Manager to fetch assets.

**Comment by lewisdonofrio** [comment]
Well after two restarts and one power down, it seems it's progressing (what is it doing, I mean it's not upgrading the linux is it??
![screenshot_20190130-105928_userland](https://user-images.githubusercontent.com/3161117/51994209-54481480-247e-11e9-90cd-3c48ee826c0b.jpg)

**Comment by lewisdonofrio** [comment]
after it finished verifiing assets it's almost back to working....what was that process and why was it needed when it was working for two months and getting playstore updates as needed???  I mean what did it do for the last two hours??
![screenshot_20190130-110318_userland](https://user-images.githubusercontent.com/3161117/51994580-1992ac00-247f-11e9-9117-9eea72d5335b.jpg)
and what is that first line in screenshot of bash prompt mean??

**Comment by lewisdonofrio** [comment]
First screenshot was the bulk of the waiting, say 1 hour or more.....I'm on wifi on asus router, when I'm downloading addon's though kodi on the note8 I get 20 megs/sec so it's not the connection that the phone has, cause I was able to click on that link you provied and even though you "use the download queue system" it downloaded just snappy like 1 second, all while the asset show was stilll going.  did it download a new repo or something (can you add "mb transfered" or something for feedback, cause I'm guessing that this will happen again.) I never knew how much I need this till it stopped working and I almost got my notebook out to keep working but I ssh'ed to another host on my network and just worked from there till this was finished.  Yes I rebooted two time while it was in first screenshot, before second I powered the device off.  once it came back it jumped to t he second screenshot.  It working now just do not know what you downloaded in that hour of downloading assets, all while other files download just fine, so I don't think your usuing queue (unless the link somehow skips that queue?)

**Comment by MatthewTighe** [comment]
Did you close the app after starting the download process when it was hanging? Did you check the progress of the download in your notification tray?

**Comment by lewisdonofrio** [comment]
I didin't see any down arrow in the tray in the first hour.....I did see it
when I clicked that link you posted as a test...hence why I'm like I do not
quite think it's using the download queue that you suggested (I am probably
incorrect but hey it's a guess)

**Comment by MatthewTighe** [comment]
Yeah, that sounds like a bug on our end. Again, I'm making some pretty sweeping changes to the code structure of the download process (though we'll still be using the download manager) that will be released to beta this week. If you experience the issue again after the update please let me know.

**Comment by lewisdonofrio** [comment]
please add backup this week....I cannot help test 18.10 or 19.04 without the ablity to backup my "Daily Setup" (just sayin if your making 'big change' this week...please) I mean thank you for all you do, without this project I'd not feel so confident about leaving my notebook home/work....

**Comment by daizu05** [comment]
This also happen to me
Im stuck in downloading resources 6/32 (kali linux)
:((

**Comment by corbinlc** [comment]
Closing as part of a repository cleanup. If this issue is still relevant, feel free to reopen it or file a fresh one.
